src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -122,9 +122,20 @@ namespace ZoneEquipmentManager {
                              Real64 &FinalTotalReturnMassFlow // Final total return air mass flow rate
     );
 
+    void CalcZoneInfiltrationFlows(EnergyPlusData &state,
+                                   int const ZoneNum,                // current zone index
+                                   Real64 &ZoneReturnAirMassFlowRate // zone total zone return air mass flow rate
+    );
```

Comment:
The zone infiltration treatment calculation used for enforcing zone air flow balancing is now refactored for flexibility of calling the function with the different flow balancing method.